pull request bbbc004

xjzhaang opened the pull request.

xjzhaang committed:
added bbbc004

sbinnee commented:
1. `file_list` and `anno_dict` do not match. 
   You need to sort files in natural order. Python built-in function `sorted` takes `key` argument. See, for example, BBBC014.

2. Delete grayscale related arguments in docstring it they are needed.
3. image should have 3 channel even if they have grayscale. 
4. Other ways to process mask?
    I see that masks have values in blue channel and you used `cv2.cvtColor()`. I think that it is unnecessary computation. It will sum and divide image by 3. You could just simply index blue channel and multiply by 255.

xjzhaang committed:
1. sorted the files based on int value
2. deleted grayscale
3. image transformed to 3 channel
4. took blue channel as mask

xjzhaang committed:
removed unused import

sbinnee committed:
Merge branch 'main' into bbbc004
